### ISSUE TITLE
v0.4.1 doesn't contains Bold and Color

### ISSUE THREAD

**Issue by zeakd** [vadimdemedes/ink#76]
Hi. Thanks for this amazing library.

Master branch (0.4.1) readme said it contains Bold and Color Component but currently npm published v0.4.1 doesn't contains those Components. 

``` js 
const { Text, Bold, Color } = require('ink');
console.log(Text, Bold, Color) // [Function: Text] undefined undefined
``` 

``` json
{
    ...
    "dependencies": {
        "ink": "^0.4.1"
    }
    ....
}
```

Thanks!

**Comment by sindresorhus** [comment]
> Master branch (0.4.1) readme said it contains Bold and Color Component but currently npm published v0.4.1 doesn't contains those Components.

Yes, because it's not published yet. Readme on master always reflects master and not the latest version. That's how most projects on GitHub works.

**Comment by zeakd** [comment]
Oh i see. Thanks
